src/routes/teams/routes.ts

```diff
@@ -0,0 +1,113 @@
+import { z, createRoute } from "@hono/zod-openapi";
+import { TeamSchema } from "../../schemas/team";
+
+export const getTeam = createRoute({
+  method: "get",
+  path: "/",
+  responses: {
+    200: {
+      description: "All teams retrieved successfully",
+      content: {
+        "application/json": {
+          schema: z.array(
+            z.object({
+              team: TeamSchema,
```

Comment:
Are you sure the data you actually send follows the defined schema?
Ensure that the returned json is like:
```
{
  data: [{}, {}, {}, ...]
}
```
where {} are teams